application-licensing-licensor/application-licensing-licensor-api/src/main/java/com/xwiki/licensing/internal/TrialLicenseGenerator.java

```diff
@@ -105,17 +103,18 @@ public void generateTrialLicense(ExtensionId extensionId)
         try {
             URL trialURL = getTrialURL(extensionId);
             if (trialURL == null) {
-                logger.info(FAILED_TO_ADD_TRIAL_LICENSE);
+                logger.debug("Failed to add trial license because the licensor configuration is not complete. "
+                    + "Check your store trial URL and owner details.");
```

Comment:
Would be good to specify the extension id.

```suggestion
                logger.debug("Failed to add trial license for [{}] because the licensor configuration is not complete. "
                    + "Check your store trial URL and owner details.", extensionId.getId());
```